pkg/security/podsecuritypolicy/provider.go

```diff
@@ -271,6 +271,21 @@ func (s *simpleProvider) ValidateContainerSecurityContext(pod *api.Pod, containe
 		allErrs = append(allErrs, field.Invalid(fldPath.Child("privileged"), *privileged, "Privileged containers are not allowed"))
 	}
 
+	procMount := sc.ProcMount()
+	allowedProcMounts := s.psp.Spec.AllowedProcMountTypes
+	if allowedProcMounts == nil {
```

Comment:
Check for empty as well? (in addition to nil)

Reply:
So a length check against 0 would catch both.